sparrow-py/pysrc/sparrow_py/expr.py

```diff
@@ -287,6 +287,9 @@ def is_not_null(self) -> "Expr":
         """Return a boolean expression indicating if the expression is not null."""
         return Expr.call("is_valid", self)
 
+    def collect(self, max: Optional[int] = None, window: Optional[Window] = None) -> "Expr":
```

Comment:
Do we want to default to "unbounded" collection, or require you specify? I think I'd prefer `max: Optional[int]` which forces the user to specify `max = None` if they want unbounded collection.

Reply:
Agreed -- that's how it should've been. Fixed